aicsimageio/tests/metadata/test_utils.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import pytest
+from ome_types import OME
+
+from aicsimageio import AICSImage
+
+from ..conftest import get_resource_full_path, host
+
+###############################################################################
+
+
+@host
+@pytest.mark.parametrize(
+    "filename",
+    [
+        # DefaultReader
+        pytest.param(
+            "example.png",
+            marks=pytest.mark.raises(execeptions=NotImplementedError),
+        ),
+        # TiffReader
+        pytest.param(
+            "s_1_t_10_c_3_z_1.tiff",
+            marks=pytest.mark.raises(execeptions=NotImplementedError),
+        ),
+        # OmeTiffReader
+        ("actk.ome.tiff"),
+        # LifReader
+        pytest.param(
+            "tiled.lif",
+            marks=pytest.mark.raises(execeptions=NotImplementedError),
+        ),
+        # CziReader
+        pytest.param(
+            "s_1_t_1_c_1_z_1.czi",
+            marks=pytest.mark.raises(execeptions=NotImplementedError),
+        ),
+        pytest.param(
+            "RGB-8bit.czi",
+            marks=pytest.mark.raises(execeptions=NotImplementedError),
+        ),
+    ],
```

Comment:
Do we have any OME-TIFFs we could add to this test, since `ome_metadata` is "implemented" for that reader?

Reply:
See [line 29](https://github.com/AllenCellModeling/aicsimageio/pull/289/files#diff-426c7a4b396b68557f25f6edddf2f7a32b68ee5f25ba2ab128fe9fdac2e762e2R29) It just isn't a `pytest.param` because it successfully completes! We only use `pytest.param` when we need to add a raises exception.